lightning/src/routing/scoring.rs

```diff
@@ -631,10 +650,11 @@ const PRECISION_LOWER_BOUND_DENOMINATOR: u64 = approx::LOWER_BITS_BOUND;
 
 /// The divisor used when computing the amount penalty.
 const AMOUNT_PENALTY_DIVISOR: u64 = 1 << 20;
+const BASE_AMOUNT_PENALTY_DIVISOR: u64 = 1 << 30;
 
 impl<L: Deref<Target = u64>, T: Time, U: Deref<Target = T>> DirectedChannelLiquidity<L, T, U> {
-	/// Returns a penalty for routing the given HTLC `amount_msat` through the channel in this
-	/// direction.
+	/// Returns a liquidity penalty for routing the given HTLC `amount_msat` through the channel in
+	/// this direction.
 	fn penalty_msat(&self, amount_msat: u64, params: &ProbabilisticScoringParameters) -> u64 {
```

Comment:
It's a little confusing here that it says this returns the "liquidity penalty" and is named `penalty_msat` but it calls `combined_penalty_msat` which says, "Computes and combines the liquidity and amount penalties."